Issue 91 Update the promotional video for the RaiderPlanner and Issue 90 Update Chat window of the Raider Planner

RichardSimmons opened the pull request:
I updated the chat window to reflect the style of the program.(Assignment 1)
I also updated promotional video to show some of the more recent changes to the program(Assignment 2)

RichardSimmons committed:
Updated help.html to include embedded video

Updated the help.html code to resolve the issue 91

RichardSimmons committed:
Updated ChatController styling

Updated Chat window of the Raider Planner to be more consistent with the styling of the overall program for issue 90

mikeknows commented on src/edu/wright/cs/raiderplanner/controller/ChatController.java:
```diff
@@ -92,8 +89,6 @@ public static void createUserMessagePane() {
 	private static void setupServerConnection() {
 		// Establish connection
 		port = 8080;
-		hostName = MenuController.getHostName();
```
This is needed from another student's pull request regarding the chat function.

RichardSimmons replied:
There was a problem with my forked repository. It should be fixed now

mikeknows commented on src/edu/wright/cs/raiderplanner/controller/ChatController.java:
```diff
@@ -59,8 +58,6 @@
 	private static PrintWriter printOutput;
 	private static InputStream incoming;
 	private static Scanner incomingMessage;
-	private static String hostName;
```
Why?

RichardSimmons committed:
Updated ChatController styling

Updated Chat window of the Raider Planner to be more consistent with the styling of the overall program for issue 90

RichardSimmons committed:
Updated help.html to include embedded video 

 Updated the help.html code to resolve the issue 91

RichardSimmons committed:
Revert "Updated ChatController styling"

This reverts commit b3cf30709b053ca666d259e1c99e48d978cddbeb.

RichardSimmons committed:
Revert "Updated help.html to include embedded video "

This reverts commit 8dcb49f8648c590fe4c7bd4c0a29b7e03973cfd6.
